src/modules/auth/server/procedures.ts

```
@@ -32,15 +32,37 @@ export const authRouter = createTRPCRouter({
+          data: {
+            email: input.email,
+            username: input.username,
+            password: input.password, // will be hashed,
+            tenants: [
+              {
+                tenant: tenant.id
+              }
+            ]
+          }
+        });
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)